core/server/error.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en" class="h-full">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LeapKit</title>
+
+    <script src="https://unpkg.com/hyperscript.org@0.9.8"></script>
```

Comment:
@tatang26 Do we need these? Looks like we could remove them.

Reply:
Great catch!, removing it